$MirOS: ports/emulators/qemu/patches/patch-slirp_misc_c,v 1.2 2005/05/03 19:30:05 tg Exp $
--- slirp/misc.c.orig	Wed Apr 27 20:51:43 2005
+++ slirp/misc.c	Tue May  3 18:04:47 2005
@@ -677,7 +677,7 @@ add_emu(buff)
 {
 	u_int lport, fport;
 	u_int8_t tos = 0, emu = 0;
-	char buff1[256], buff2[256], buff4[128];
+	char buff1[257], buff2[257], buff4[128];
 	char *buff3 = buff4;
 	struct emu_t *emup;
 	struct socket *so;
